src/Pages/Onboarding1.jsx

```diff
@@ -0,0 +1,201 @@
+                    He/Him
+                  </label>
+                </div>
+                <div style={styles.radiobutton}>
+                  <input
+                    className="radioButton"
+                    type="radio"
+                    name="pronouns"
+                    id="sheher"
+                  />
+                  <label className="checkboxLabel" style={styles.checkboxLabel} htmlFor="sheher">
+                    She/Her
+                  </label>
+                </div>
+                <div style={styles.radiobutton}>
+                  <input
+                    className="radioButton"
+                    type="radio"
+                    name="pronouns"
+                    id="theythem"
+                  />
+                  <label className="checkboxLabel" style={styles.checkboxLabel} htmlFor="theythem">
+                    They/Them
+                  </label>
+                </div>
+                <div style={styles.radiobutton}>
+                  <input
+                    className="radioButton"
+                    type="radio"
+                    name="pronouns"
+                    id="otherpro"
+                  />
+                  <label className="checkboxLabel" htmlFor="otherpro">
+                    Other
+                  </label>
+                </div>
+              </div>
+            </div>
+
+            <div className="ProfilePicture" style={styles.infoBlock}>
+              <div style={styles.boldparagraph}>Profile picture:</div>
+              <SecondaryButton>Upload Picture</SecondaryButton>
+            </div>
+
+            <div className="profileBio" style={styles.infoBlock}>
+              <div style={styles.boldparagraph}>Profile bio:</div>
+              <textarea style={styles.biotext} id="bioinfo" rows="5" cols="33" placeholder="Write your bio here..."/>
+            </div>
+          </div>
+          <Link to="/onboarding2" style={styles.nextButton}>
+            <PrimaryButton>Next</PrimaryButton>
+          </Link>
+        </div>
+        <HorizontalLine />
+        </form>
+      </div>
+  );
+};
+
+export default Onboarding1;
+
+//Styling
+const styles = {
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    minHeight: "100vh",
+    height: '100vh',
+    width: '100vw',
+    position: 'relative',
+  },
+  form: {    
+    backgroundColor: "rgba(245, 245, 245, 1)",
+    borderRadius: "20px",
+    padding: "45px",
+    width: "550px",
+    position: "relative",
+    textAlign: "left",
+    boxShadow: "0 4px 8px rgba(0, 0, 0, 0.1)",
+  },
+  mainTitle: {
+    fontSize: "2em",
+    margin: "0 0 10px 0",
+    color: "#35415D",
+    fontFamily: "Inter, sans-serif",
+    fontWeight: "bold",
+  },
+  paragraph: {
+    fontSize: "1em",
+    margin: "10px 0",
+    lineHeight: "1.5",
+    color: "#333",
+  },
+  boldparagraph: {
+    fontSize: "1em",
```

Comment:
please decided on size values. px, rem, em are all used in this file. Clean up your styling when converting to styled components